src/bap_mc/bap_mc.ml

```diff
@@ -64,6 +64,13 @@ let print_kinds insn =
   List.map ~f:sexp_of_kind |>
   List.iter ~f:(printf "%a@." Sexp.pp)
 
+let print_insn_oplen oplen_bases mem =
+  let len = (Memory.length mem) in
+  (List.iter oplen_bases
+	     ~f:(function
+		  | `hex -> printf (format_of_string "%x\n") len
```

Comment:
there is no need to call `format_of_string` on string literals when using `printf`. Constant string literals are already of type format when needed (they are overloaded). Moreover, it is better to use `%#x` specifier, since without it numbers will be printed without leading `0x`, that is very confusing. So, the better version would be:
```ocaml
printf "%#x" len
```
Also, all parenthesis in this function are excessive.